### ISSUE TITLE
0.16.1 bundle not including main entries

### ISSUE THREAD

**Issue by ghempton** [systemjs/builder#766]
This previously worked in an earlier version of SystemJS before this was fixed: https://github.com/systemjs/systemjs/issues/1552

Here is an example of the issue: https://github.com/ghempton/systemjs-builder-test/tree/missing-main

Looks like importing main entries in configured packages is not including them in the bundle.

Moreover, this might be a more generic SystemJS issue as when I have the following configuration:

```
SystemJS.config({
  paths: {
    "npm:": "jspm_packages/npm/",
    "app/": "src/"
  },
  browserConfig: {
    "baseURL": "/"
  },
  packages: {
    "app": {
      "format": "system",
      "main": "index.js"
    }
  }
});
```

Running `SystemJS.normalizeSync('app')` returns `/app` rather than `/app/index.js` as I would expect.

**Comment by guybedford** [comment]
Thanks again for posting, yes it looks like it was a regression of that. I've added a fix in https://github.com/systemjs/systemjs/commit/d1bdd5d150b2ce49ccb877bf8ec0858dbb7fd63a.

**Comment by guybedford** [comment]
Released in SystemJS 0.20.4.
